### PR TITLE
Added firebase

### PR DESCRIPTION
**Pull request by anw98**
Set up firebase firestore and saved the card data in the firestore. Used redux to store firestore data and used saga library to handle side effects.

### CODE REVIEW

**Comment by anw98** [comment]
Set up firebase firestore and saved the card data in the firestore. Used redux to store firestore data and used saga library to handle side effects.

File: src/actions/addCard.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { CardTravelSharp, Store } from "@material-ui/icons";
+import {collection, addDoc} from "firebase/firestore";
+import db from '../Utils/firebaseConfig';
+
+export default async function addCard(card){
+    await addDoc(collection(db,"cards"),card);
```
Comment (JinapriyaM) [inline src/actions/addCard.js]: Only define relevant action here. Move this to a new saga.

**Review by JinapriyaM — CHANGES_REQUESTED** [review]

File: src/actions/addCard.js (hunk 1 above)
Comment (anw98) [reply src/actions/addCard.js]: Moved addCard to a new saga

**Comment by anw98** [comment]
Moved addCard to a new saga

**Pull request closed without merging**